Library/Formula/slacknimate.rb

```diff
@@ -0,0 +1,24 @@
+class Slacknimate < Formula
+  desc "text animation for Slack messages"
+  homepage "https://github.com/mroth/slacknimate"
+  url "https://github.com/mroth/slacknimate/archive/v1.0.0.tar.gz"
+  sha256 "7d495e2233a379f3ffffb2445f4b41b4306e52049d71f4c4c873786bb4048f45"
+  head "https://github.com/mroth/slacknimate.git"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/mroth"
+    ln_s buildpath, buildpath/"src/github.com/mroth/slacknimate"
+    system "glide", "install"
```

Comment:
I understand your concerns, but `go_resource`s solve a problem that `glide` doesn’t: caching. I can run `brew fetch slacknimate` once then get my train and tweak the formula without internet connection because `brew install slacknimate` won’t download any external resource. I can `brew rm` and `brew install` again without worrying because resources are cached.

The caching issue is on our (long) TODO list and some formulae do download resources when installing, like Node/Java/Haskell stuff because e.g. with Node we’d need hundreds of `resource do` blocks even for simple formulae.

A solution would be to use the `vendor` directory introduced by Go 1.5 and now supported by default since Go 1.6; but that requires work on *your* (read: upstream) side. We don’t have any solution other than `go_resource`s on our side for now.

(the essay was posted on our Slack but I haven’t read it entirely yet :sweat_smile: )